delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/storages/StoragesStatistics.scala

```diff
@@ -1,49 +1,92 @@
 package ch.epfl.bluebrain.nexus.delta.plugins.storage.storages
 
-import ch.epfl.bluebrain.nexus.delta.plugins.storage.storages.model.StorageRejection.{StorageFetchRejection, StorageNotFound}
-import ch.epfl.bluebrain.nexus.delta.plugins.storage.storages.model.StorageStatsCollection
-import ch.epfl.bluebrain.nexus.delta.plugins.storage.storages.model.StorageStatsCollection.StorageStatEntry
+import akka.http.scaladsl.model.Uri.Query
+import ch.epfl.bluebrain.nexus.delta.kernel.utils.ClasspathResourceUtils.ioJsonObjectContentOf
+import ch.epfl.bluebrain.nexus.delta.plugins.elasticsearch.EventMetricsProjection
+import ch.epfl.bluebrain.nexus.delta.plugins.elasticsearch.client.ElasticSearchClient
+import ch.epfl.bluebrain.nexus.delta.plugins.storage.storages.model.StorageRejection.StorageFetchRejection
+import ch.epfl.bluebrain.nexus.delta.plugins.storage.storages.model.StorageStatEntry
 import ch.epfl.bluebrain.nexus.delta.rdf.IriOrBNode.Iri
+import ch.epfl.bluebrain.nexus.delta.sdk.http.HttpClient.HttpResult
 import ch.epfl.bluebrain.nexus.delta.sdk.model.IdSegment
+import ch.epfl.bluebrain.nexus.delta.sdk.syntax._
 import ch.epfl.bluebrain.nexus.delta.sourcing.model.ProjectRef
+import com.typesafe.scalalogging.Logger
+import io.circe.{Json, JsonObject}
 import monix.bio.{IO, UIO}
 
 trait StoragesStatistics {
 
   /**
-    * Retrieve the current statistics for all storages
-    */
-  def get(): UIO[StorageStatsCollection]
-
-  /**
-    * Retrieve the current statistics for storages in the given project
-    */
-  def get(project: ProjectRef): UIO[Map[Iri, StorageStatEntry]]
-
-  /**
-    * Retrieve the current statistics for the given project
+    * Retrieve the current statistics for a given storage in the given project
     */
   def get(idSegment: IdSegment, project: ProjectRef): IO[StorageFetchRejection, StorageStatEntry]
 
 }
 
-// TODO implements storage statistics
 object StoragesStatistics {
 
-  def apply(): StoragesStatistics = new StoragesStatistics {
+  implicit private val classLoader: ClassLoader = getClass.getClassLoader
+  implicit private val logger: Logger           = Logger[StoragesStatistics]
+
+  private val storagesAggByIdAndProjectQuery =
```

Comment:
As the query is not very complicated and to avoid these replaces, you should be able to use `circe-literal` that would make it look like this:
```scala
import io.circe.literal._

val query =  json"""{ ... }"""
```
We do mostly the same thing with sql queries in the end
It should also increase readability because we would not have to jump from one file to the other